lib/mayaUsd/render/vp2RenderDelegate/selectability.h

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright 2021 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef HD_VP2_SELECTABILITY_H
```

Comment:
This service is not tight to VP2RenderDelegate and can be used by our other rendering code paths (e.g. MayaToHydra). Let's move it outside of VP2RenderDelegate and validate that MayaToHydra can leverage it too.